tests/countries/test_dr_congo.py

```diff
@@ -0,0 +1,248 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.dr_congo import DRCongo, CD, COD
+from tests.common import CommonCountryTests
+
+
+class TestDRCongo(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(DRCongo, years=range(1980, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(DRCongo, CD, COD)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(DRCongo(years=1979))
+
+    def test_new_years_day(self):
+        self.assertHolidayName("Nouvel an", (f"{year}-01-01" for year in range(1980, 2050)))
+
+    def test_martyrs_day(self):
+        name = "Martyrs de l'indépendance"
+        self.assertHolidayName(name, (f"{year}-01-04" for year in range(2015, 2050)))
+        self.assertNoHolidayName(name, range(1980, 2015))
+
+        dt = ("2015-01-03",)
+        self.assertHoliday(dt)
+        self.assertNoNonObservedHoliday(dt)
```

Comment:
```suggestion
        obs_dt = ("2015-01-03",)
        self.assertHolidayName(f"{name} (observé)", obs_dt)
        self.assertNoNonObservedHoliday(obs_dt)
```
And same for all holidays.